infrastructure/transform/feature-data-store.tf

```diff
@@ -219,3 +219,21 @@ resource "azurerm_private_endpoint" "sql_server_features_pe" {
     private_dns_zone_ids = [azurerm_private_dns_zone.sql.id]
   }
 }
+
+resource "azuread_group" "ad_group_apps" {
+  display_name     = "${var.naming_suffix} flowehr-apps"
+  owners           = [data.azurerm_client_config.current.object_id]
+  security_enabled = true
+}
+
+resource "azuread_group" "ad_group_developers" {
+  display_name     = "${var.naming_suffix} flowehr-developers"
+  owners           = [data.azurerm_client_config.current.object_id]
+  security_enabled = true
+}
+
+resource "azuread_group" "ad_group_data_scientists" {
+  display_name     = "${var.naming_suffix} flowehr-data-scientists"
+  owners           = [data.azurerm_client_config.current.object_id]
+  security_enabled = true
+}
```

Comment:
if these groups get used in transform+apps+tre then I would make them in core?

Reply:
Not sure I agree, think it should happen just after database creation so seems like the right place

Reply:
is their only purpose going to be for feature store access? a group full of app devs would be useful for other things(?)
(feel free to ignore - don’t feel strongly)

Reply:
Potentially - I reckon we move when that becomes the case, right now there aren't any other use cases I'm aware of, but fair point!